ome-tiff-tiler-manifest.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "pattern": "output_tiff_pyramids/*.ome.tif*",
+    "description": "Multi-resolution OME-TIFF file for visualiztion."
+  },
+  {
+    "pattern": "output_tiff_pyramids/*.n5",
+    "description": "Multi-resolution n5 folder for data analysis/visualiztion locally (can be loaded into a Zarr file)."
```

Comment:
I think Matt is just applying these patterns to files, not directories? So perhaps...
```suggestion
    "pattern": "output_tiff_pyramids/.*\\.n5/.*",
    "description": "Content of multi-resolution n5 folder for data analysis/visualization locally (can be loaded into a Zarr file)."
```

Reply:
@mruffalo Does your system know to recurse on folders?  This folder contains many levels.